scripts/copy_db_from_pod_to_local.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+# Database copy helper
+#
+#   Connect to an openshift pod, dump a postgresql database and copy it locally.
+#
+# Usage:
+#
+#   PROJECT=OPENSHIFT_PROJECT_NAME POD=OPENSHIFT_POD DATABASE=POSTGRESQL_DATABASE ${THIS_FILE}
+#
+#   Assumes you have openshift command line tools installed and are logged in.
+#
+# Examples:
+#   
+#   Copy database from a pod in production.
+#   PROJECT=auzhsi-prod POD=patroni-wps-api-prod-2 DATABASE=wps-api-prod ./restore_db.sh
```

Comment:
Did you forget to rename `restore_db.sh`?

Reply:
I did indeed.